learn/getting_started/quick_start.mdx

```diff
@@ -1,35 +1,37 @@
 # Quick start
 
-This quick start will walk you through setting up Meilisearch, adding documents, performing your first search, using the search preview, adding a search bar, and securing your instance.
+This quick start walks you through installing Meilisearch, adding documents, and performing your first search.
 
-You will need:
+To follow this tutorial you need:
 
-- A [command line](https://www.learnenough.com/command-line-tutorial#sec-running_a_terminal) for installation
-- Some way to interact with Meilisearch afterwards (for example, [cURL](https://curl.se) or one of our [SDKs](/learn/what_is_meilisearch/sdks))
-
-Let's get started!
+- A [command line terminal](https://www.learnenough.com/command-line-tutorial#sec-running_a_terminal)
+- [cURL](https://curl.se) 
+- Optional: one of the official Meilisearch [SDKs](/learn/what_is_meilisearch/sdks) if you want to follow along in a specific programming language
 
 ## Setup and installation
 
-We'll start with downloading and installing Meilisearch:
+First, you need to download and install Meilisearch. This command installs Meilisearch in your local machine:
 
 ```bash
 # Install Meilisearch
 curl -L https://install.meilisearch.com | sh
-
-# Launch Meilisearch
-./meilisearch
 ```
 
-You have the option to install Meilisearch locally or deploy it over a cloud service. Learn more about the other installation options in our [installation guide](/learn/getting_started/installation).
+The rest of this guide assumes you are using Meilisearch locally, but you may also use Meilisearch over a cloud service such as [Meilisearch Cloud](https://www.meilisearch.com/pricing). 
+
+Learn more about other installation options in the [installation guide](/learn/getting_started/installation).
 
-<Capsule intent="note">
-If you started Meilisearch with a master key or using Meilisearch Cloud, you will need to add the [`-H Authorization: Bearer API_KEY` header](/reference/api/overview#authorization) to all your curl commands.
-</Capsule>
 
 ### Running Meilisearch
 
-On successfully running Meilisearch, you should see the following response:
+Next, launch Meilisearch by running the following command in your terminal:
+
+```bash
+# Launch Meilisearch
+./meilisearch --master-key="aSampleMasterKey"
+```
+
+You should see something a bit like this in response:
```

Comment:
A bit sounds like users may get different responses. Wouldn't "you should see the following response" or something be better?

Reply:
I sort of want to be imprecise here because the exact message may look different due to user setup (the Meilisearch ASCII art may get messed up depending on your terminal size) or to minor changes made by the engine team (for example, users nowadays see `Config file path` before `Database path`.